### PR TITLE
Add a load of typos from Buildbot

### PR DESCRIPTION
**Pull request by peternewman**
From https://github.com/buildbot/buildbot, mostly from https://github.com/buildbot/buildbot/blob/master/master/docs/spelling_wordlist.txt

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -10707,8 +10742,10 @@ sortings->sorting
 sortlst->sortlist
 sortner->sorter
 sortnr->sorter
+sortrage->storage
 soruce->source
 soruces->sources
+sotrage->storage
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: Can also be `shortage`

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -10707,8 +10742,10 @@ sortings->sorting
 sortlst->sortlist
 sortner->sorter
 sortnr->sorter
+sortrage->storage
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: Can also be `shortage`

Comment (peternewman) [reply codespell_lib/data/dictionary.txt]: Done.

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (peternewman) [reply codespell_lib/data/dictionary.txt]: Done, although this one feels a bit of a leap to me.

**Pull request merged**